packages/apollo-gateway/src/__tests__/integration/value-types-interfaces.test.ts

```diff
@@ -0,0 +1,6247 @@
+import gql from 'graphql-tag';
```

Comment:
Are you going to drop this file before submitting PR to master?

Reply:
The problem that we couldn't find an exact case causing the error, so we planned to:
- merge it as is 
- later to minimize this query to contain types that actually raising the error in follow up ticket
- Submit PR fix to original repo


Because it's stoping staff portal for running gateway we want to merge it ASAP 🌝